src/three-components/TextureUtils.ts

```diff
@@ -424,18 +424,26 @@ void main() {
 
   async dispose() {
     for (const environmentMapLoads of this[$environmentMapCache].values()) {
-      const environmentMap = await environmentMapLoads;
-      if (environmentMap != null) {
-        environmentMap.dispose();
+      try {
+        const environmentMap = await environmentMapLoads;
+        if (environmentMap != null) {
+          environmentMap.dispose();
+        }
+      } catch (e) {
+        // Suppress errors
```

Comment:
What errors are we getting?

Reply:
If a promise is rejected (for example. say a file fails to load), the promise is rejected for all time. If you later `await` that promise, it will throw an error.